src/DefaultUser.php

```diff
@@ -26,15 +26,19 @@ final class DefaultUser implements UserInterface
     private $identity;
 
     /**
-     * @var string[]
+     * @var array<mixed>
```

Comment:
TBH: this is less accurate than before.
If this was a list of strings before, please consider adding `@psalm-var list<string>`.
If this was a map before, consider adding `psalm-var array<string,string>`

Reply:
@boesing, recheck please.